drivers/gles2/rasterizer_scene_gles2.cpp

```diff
@@ -3439,6 +3445,10 @@ void RasterizerSceneGLES2::render_scene(const Transform &p_cam_transform, const
 					clear_color = Color(0.0, 1.0, 0.0, 1.0);
 				}
 			} break;
+			case VS::ENV_BG_CANVAS: {
+				// use screen copy as background
+				_copy_texture_to_buffer(storage->frame.current_rt->copy_screen_effect.color, current_fb);
+			}	break;
 			default: {
 				// FIXME: implement other background modes
```

Comment:
I think that `FIXME` might be solved now, at least GLES3 also seems to handle only three modes in this switch.